src/app/App.vue

```diff
@@ -78,7 +78,7 @@ const isLoading = ref(store.state.globalLoading)
 const routeKey = computed(() => route.meta.shouldReRender ? route.path : 'NONE')
 const shouldShowAppError = computed(() => store.state.config.status !== 'OK')
 const shouldSuggestOnboarding = computed(() => store.getters['onboarding/showOnboarding'])
-const shouldShowNotificationManager = computed(() => !shouldSuggestOnboarding.value && store.getters['notifications/amountOfActions'] > 0)
+const shouldShowNotificationManager = computed(() => store.getters['notifications/amountOfActions'] > 0)
```

Comment:
why are we not taking `shouldSuggestOnboarding.value` into account any more?

Reply:
I don’t know why it was there in the first place because it’s not relevant here.

The notification manager is only invoked by user interaction from the top bar so this doesn’t mean to show the modal right away, it just _allows_ it do be shown in the first place (i.e. when the user clicks on the notification icon in the top bar).